### ISSUE TITLE
Internal error on video trigger

### ISSUE THREAD

**Issue by ggalan87** [ossia/score#1377]
Hello. I encountered the following error while playing a simple pipeline with a single video and a trigger enabled in the end. The message pops up when the trigger (or end of video file) in the timeline is reached.

![Screenshot from 2022-03-25 12-19-59](https://user-images.githubusercontent.com/3819676/160139936-f6efd702-c15c-4ed2-915d-1487f6fdd198.png)

The exact same project file produces this error for versions 3.0.4-3.0.6 that I tested, while in version 3.0.3 it's OK.

As long as the error is triggered, the video window gets stuck and no video output is produced if restarted. After the first error is triggered, the same message can be triggered again in the setting for the Graphics API, e.g. changing from OpenGL to Vulkan as shown below. I think that it falls in a weird graphics error state.

![Screenshot from 2022-03-25 12-31-59](https://user-images.githubusercontent.com/3819676/160139974-afcff9fc-05e5-4dea-b5e1-4e31f1e729c5.png)

OS: Ubuntu 21.04

**Comment by jcelerier** [comment]
thanks, checking that asap !

**Comment by jcelerier** [comment]
hummm weird, here it doesn't do that. Could you send my your score file and ideally your video file through e.g. wetransfer ?

**Comment by ggalan87** [comment]
> hummm weird, here it doesn't do that. Could you send my your score file and ideally your video file through e.g. wetransfer ?

In fact in another setup I had (score 3.0.5) I thought that it was the video, because I downloaded one from youtube, got this error, but then I downloaded from the same source as you had in the tutorial (beeple videos) and voila it worked. I currently don't have access in that setup, maybe in few days to test the exact same score file I attach.

So the video I tested is this: https://vimeo.com/238083470 (original resolution) and the score file is here [untitled.zip](https://github.com/ossia/score/files/8352244/untitled.zip). It's not the same video as in the screenshot but the same happens.

Maybe it is a transient issue, or maybe it has to do with the "random" placement of the video on the timeline?
